main.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  version = "~> 0.11"
+  required_version = "~> 0.11"
```

Comment:
Should this constraint not be `~> 0.12.0` since you're using the new syntax?

Reply:
Yeah, this was just the stock state after running the 0.12upgrade command and fixing the couple things that were broken. I've since updated it.